src/vue/backend/src/views/Warnings.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
The content should be passed to the feedback component as a slot, that way it can be completely customizable and reusable. You can make a slot that has the same styles